lib/tapioca/generators/base_generator.rb

```diff
@@ -0,0 +1,110 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "active_support/all"
```

Comment:
Is this needed? I see underscore but [generator has its own underscore](https://github.com/Shopify/tapioca/pull/481) and I think `require "pathname"` can take care of the path stuff.